frontend/src/actions/authActions.js

```diff
@@ -0,0 +1,109 @@
+import axios from 'axios';
+
+import {returnErrors} from './errorActions';
+
+import {
+    USER_LOADED,
+    USER_LOADING,
+    AUTH_ERROR,
+    LOGIN_SUCCESS,
+    LOGIN_FAIL,
+    LOGOUT_SUCCESS,
+    REGISTER_SUCCESS,
+    REGISTER_FAIL,
+} from './types';
```

Comment:
we had action types to store those, if you feels more comfortable having action types inside action folder then remove the old `actionType` folder

Reply:
yeah I felt like one file is good, all of our actions live in one file